controllers/provider-openstack/pkg/webhook/controlplane/ensurer.go

```diff
@@ -57,9 +57,9 @@ func (e *ensurer) EnsureKubeAPIServerDeployment(ctx context.Context, dep *appsv1
 	ps := &template.Spec
 	if c := controlplane.ContainerWithName(ps.Containers, "kube-apiserver"); c != nil {
 		ensureKubeAPIServerCommandLineArgs(c)
-		ensureVolumeMounts(c)
+		//ensureVolumeMounts(c)
```

Comment:
Why is this commented out?

Reply:
The kube-api server doesn't need the cloud config anymore.

Reply:
Why not? We don't have CSI yet, and it still labels the PVs with region/zone info.